pkg/toolsets/openshift/toolset.go

```diff
@@ -0,0 +1,31 @@
+package openshift
+
+import (
+	"slices"
+
+	"github.com/containers/kubernetes-mcp-server/pkg/api"
+	internalk8s "github.com/containers/kubernetes-mcp-server/pkg/kubernetes"
+	"github.com/containers/kubernetes-mcp-server/pkg/toolsets"
+)
+
+type Toolset struct{}
+
+var _ api.Toolset = (*Toolset)(nil)
+
+func (t *Toolset) GetName() string {
+	return "openshift-core"
```

Comment:
Let's rename this to be more specific like `openshift-node-debug`

Reply:
how about `openshift-debug`?  I was thinking of adding `oc rsync` kind of tool too, which may not be limited to just node related debugging.